modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/js/blogs.js

```diff
@@ -198,6 +200,44 @@ AUI.add(
 						)
 					);
 
+					var titleEditor = instance.one(
+						'#' + instance.ns('titleEditor')
+					);
+
+					if (titleEditor) {
```

Comment:
This feels very boilerplatish and repetitive... I think we used to have an auto-sizeable textarea in AlloyUI or DXP but couldn't find any traces of it in a quick search.

Not sure if that's a good pattern, but maybe we could follow the "JQuery approach" and provide some kind of API like:

```html
<textarea class="liferay-size-to-fit-textarea" ... />
```

Either this or add the feature into the `aui:input` tag, but this feels like a generic and complex enough feature to go duplicating it over and over...

/cc @wincent, thoughts on this?

Reply:
Found it -> https://github.com/liferay/alloy-ui/blob/master-deprecated/src/aui-autosize-deprecated/js/aui-autosize-deprecated.js

Reply:
Why can't we just use the `autoSize` attribute of the tag?

https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/taglib/aui/input/page.jsp#L358-L362

Reply:
If you can reuse an existing implementation, great. If you can't though, I would think that the `keydown` event wouldn't be the only one you'd want to listen to (for example, you can delete text with a mouse, you can paste etc); would probably want the [input](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/input_event) event.

Reply:
Ok I will use that. I hadn't realized it would provide this functionality